tests/test_tags.py

```diff
@@ -655,6 +655,155 @@ def test_linux_platforms_manylinux2014(monkeypatch):
     assert platforms == expected
 
 
+def test_linux_platforms_manylinux2014_armhf_abi(monkeypatch):
+    monkeypatch.setattr(
+        tags, "_is_manylinux_compatible", lambda name, _: name == "manylinux2014"
+    )
+    if platform.system() != "Linux" or platform.machine() != "armv7l":
```

Comment:
Similar comment as on a previous commit, since this runs on every platform I would just make the `monkeypatch.setattr` unconditional here and on the other new tests.

Reply:
done